stripe/src/main/java/com/stripe/android/PaymentSessionConfig.kt

```diff
@@ -75,8 +78,8 @@ data class PaymentSessionConfig internal constructor(
 
         /**
          * @param shippingMethodsRequired whether a [com.stripe.android.model.ShippingMethod]
-         * should be required. If it is required, a screen with a
-         * [com.stripe.android.view.SelectShippingMethodWidget] can be shown to collect it.
+         * should be required. If it is required, a screen with a [SelectShippingMethodWidget]
+         * can be shown to collect it.
```

Comment:
> This method sets whether or not the PaymentSession requires the user to select a shipping method (Overnight, Standard Shipping, etc). The default value is true. You must set these values later.

Also, "a screen ... can be shown to collect it" -> "a screen *is* shown to collect it"?